src/pages/Home/Home.scss

```diff
@@ -48,6 +48,17 @@
     margin-right: 200px;
     margin-top: 50px;
     width: 50px;
+    p {
```

Comment:
Great solution. I see a lot of duplicate code, can you please add a class to the <p> tag, so that you can reuse the same style on all of these p tags instead of manually duplicate the same code on each <p> tag. Thank you Shimi.